backend/src/openarchiefbeheer/destruction/tests/test_serializers.py

```diff
@@ -122,6 +124,74 @@ def test_create_destruction_list(self):
             '[2024-05-02T16:00:00+02:00]: Destruction list "A test list" created by user record_manager.',
         )
 
+    def test_destruction_list_with_short_procedure_requires_multiple_reviewers(self):
+        reviewer = UserFactory.create(
+            username="reviewer1",
+            email="reviewer1@oab.nl",
+            role__can_review_destruction=True,
+        )
+        record_manager = UserFactory.create(
+            username="record_manager", role__can_start_destruction=True
+        )
+        zaak = ZaakFactory.create()
+        request = factory.get("/foo")
+        request.user = record_manager
+        config = ArchiveConfig.get_solo()
+        config.zaaktypes_short_process = [zaak.zaaktype]
+        config.save()
```

Comment:
It is better to avoid the get_solo calls and modifying the actual instance of the solo model because this can break test isolation due to the solo cache. In general I try to use a patch for the get_solo, for example: https://github.com/maykinmedia/open-archiefbeheer/blob/main/backend/src/openarchiefbeheer/destruction/tests/test_models.py#L215